app/controllers/emails_controller.rb

```diff
@@ -15,7 +15,7 @@ def ask_webmaster_question
     @email = @user.email if @user
   end
 
-  def ask_user_question
+  def ask_user_question # rubocop:disable Metrics/AbcSize
```

Comment:
I think this is something for another PR. But just in case:
We could get rid of the ABC offense by refactoring UserMail.build to take kwargs and eliminating unnecessary local variable assignments. 
See https://www.pivotaltracker.com/story/show/184120794 and https://www.pivotaltracker.com/story/show/184124094

Reply:
I like this idea, and I hope you will tackle it! 

I'm also wanting to convert `assert_request` and the `either_requires_either` feeder methods into kwargs.